lib/taskana-core-test/src/test/java/acceptance/task/requestreview/RequestReviewAccTest.java

```diff
@@ -0,0 +1,172 @@
+  ObjectReference defaultObjectReference;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup(ClassificationService classificationService, WorkbasketService workbasketService)
+      throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification().buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+
+    defaultObjectReference = DefaultTestEntities.defaultTestObjectReference().build();
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_RequestReview_WhenTaskIsClaimed() throws Exception {
+    Instant now = Instant.now();
+    Task task = createTaskClaimedByUser("user-1-1").buildAndStore(taskService);
+    Task result = taskService.requestReview(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ForceRequestReview_WhenTaskIsClaimedByDifferentUser() throws Exception {
+    Task task = createTaskClaimedByUser("user-1-2").buildAndStore(taskService);
+    Instant now = Instant.now();
+    Task result = taskService.forceRequestReview(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ForceRequestReview_WhenTaskIsInReviewByDifferentUser() throws Exception {
+    Task task =
+        createTaskClaimedByUser("user-1-2").state(TaskState.IN_REVIEW).buildAndStore(taskService);
+    Instant now = Instant.now();
+    Task result = taskService.forceRequestReview(task.getId());
+
+    assertThat(result.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
+    assertThat(result.getOwner()).isNull();
+    assertThat(result.getModified()).isAfterOrEqualTo(now);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_RequestingReviewAndTaskIsClaimedByDifferentOwner()
+      throws Exception {
+    Task task = createTaskClaimedByUser("user-1-2").buildAndStore(taskService);
+    ThrowingCallable call = () -> taskService.requestReview(task.getId());
+
+    assertThatThrownBy(call)
+        .isInstanceOf(InvalidOwnerException.class)
+        .hasMessageContaining("User 'user-1-1' is not owner");
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @TestFactory
+  Stream<DynamicTest> should_ThrowException_When_RequestingReviewAndTaskIsNotClaimed() {
```

Comment:
should we create a counterpart test where this behavior is possible via "force"?